src/openads/domain/position/PositionNotFoundException.js

```diff
@@ -0,0 +1,5 @@
+export default class PositionNotFoundException extends Error {
+  constructor ({id}) {
+    super(`Position ${id} not found.`)
```

Comment:
This won't work as expected I think.
As we saw, calling to super will not initialize the message, name and stack properties of the error

Reply:
Done: custom name is added to both domain exceptions. Also, name and stack are checked in tests.

Reply:
Test in JS bin here: https://jsbin.com/savulidopa/edit?js,console

Reply:
Not valid man :)

The JSBin is using Javascript instead of ES6/Babel (don't know why it's working)
But if you use the correct profile for the bin:

![captura de pantalla 2018-02-15 a las 18 52 10](https://user-images.githubusercontent.com/20399660/36272291-60698858-1281-11e8-93f8-f529870a81f4.png)

Reply:
As we agreed, I refactored both custom exceptions to set name, stack and message properties avoiding to pass any parameter to the super constructor.